### ISSUE TITLE
Fix incorrect examples

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#238]
The schema currently contains errors, because some examples list multiple values:

```
media-table-schema.json
✔︎ valid JSON
✕ valid Table Schema, errors:
	 - Schema is not valid: example value for field "filePath" is not valid

deployments-table-schema.json
✔︎ valid JSON
✕ valid Table Schema, errors:
	 - Schema is not valid: example value for field "setupBy" is not valid

observations-table-schema.json
✔︎ valid JSON
✕ valid Table Schema, errors:
	 - Schema is not valid: example value for field "classifiedBy" is not valid
```

**Comment by peterdesmet** [comment]
@kbubnicki the currently listed example for filePath is https://trapper.org/storage/resource/media/259024/file/, but that results in an invalid response. Do you have a better example, or should I use one from Agouti?

**Comment by peterdesmet** [comment]
Also, for `classifiedBy` I think it's better to have a value with a version. Currently it's "Megadetector". Is there something like "Megadetector v1"?

**Comment by peterdesmet** [comment]
Fixed in #239, used `MegaDetector V5` per documentation I found online.
